pkg/limayaml/validate.go

```diff
@@ -303,6 +303,16 @@ func Validate(y LimaYAML, warn bool) error {
 	if err := validateNetwork(y, warn); err != nil {
 		return err
 	}
+
+	switch *y.ExcludeFromBackup {
+	case ExcludeFromBackupAll, ExcludeFromBackupDisks, ExcludeFromBackupNone:
+	default:
+		return fmt.Errorf("field `excludeFromBackup` must be %q, %q or %q; got %q", ExcludeFromBackupAll, ExcludeFromBackupDisks, ExcludeFromBackupNone, *y.ExcludeFromBackup)
```

Comment:
I know it isn't necessary, but personally I prefer to always use the oxford comma:

```suggestion
		return fmt.Errorf("field `excludeFromBackup` must be %q, %q, or %q; got %q", ExcludeFromBackupAll, ExcludeFromBackupDisks, ExcludeFromBackupNone, *y.ExcludeFromBackup)
```